plugins/user.py

```diff
@@ -358,6 +360,10 @@ def user_edit_profile(username):
                          ('mail', form.mail),
                          ('userAccountControl', form.uac_flags)]
 
+        if user['cUJAEPersonExternal'] == 'TRUE':
```

Comment:
The key Error commented previously seems to be related to this line.

Reply:
I tested the code on my local environment and it worked fine. It might be incompatibilities between the branches. I had to clone the git code in order to be able to create the PR